source/draw/README.md

```diff
@@ -17,6 +17,11 @@ The destination is similar to the viewport by defining where to draw. The differ
 ## Transform
 Transform is the inverse camera matrix applied after the destination scaling.
 
+## FlatTransform
+FlatTransform is applied after the projection in 2D.
```

Comment:
We want to make things abstract so it should not be mixed with the projection matrix. It is a 2D operation in its own 2D space.